src/test/run-make-fulldeps/coverage-reports/Makefile

```diff
@@ -106,6 +106,7 @@ endif
 
 	# Postprocess the profiling data so it can be used by the llvm-cov tool
 	"$(LLVM_BIN_DIR)"/llvm-profdata merge --sparse \
+			"$(TMPDIR)"/$@.profraw \
 			"$(TMPDIR)"/$@-*.profraw \
```

Comment:
Not sure if this make/shell syntax will fail if no such files exist? In that case I might have to replicate the shell-for from down below.

TBH, I haven’t run this yet, its quite a pain to make this work on windows :-(